af/firefox-ios.xliff

```diff
@@ -485,31 +485,24 @@
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
     </header>
     <body>
+      <trans-unit id="A modern mobile browser from Mozilla, the non-profit committed to a free and open web.">
```

Comment:
These IDs are not correct. They should not be the strings, but instead be actual identifiers.